lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -1014,7 +1015,6 @@ def alerts(self) -> dict:
                 try:
                     scrape_metadata = json.loads(relation.data[relation.app]["scrape_metadata"])
                     identifier = JujuTopology.from_dict(scrape_metadata).identifier
```

Comment:
we should move this down, out of the try/except block (it's not going to raise keyerror anyway)

Reply:
(This lib belongs to prom, not am.)

Reply:
oh! silly me, didn't notice :)